pkgs/development/python-modules/dns-lexicon/default.nix

```
@@ -0,0 +1,42 @@
+{ buildPythonPackage
+, fetchPypi
+, poetry-core
+, beautifulsoup4
+, cryptography
+, importlib-metadata
+, pyyaml
+, requests
+, tldextract
+, lib
+}:
+
+buildPythonPackage rec {
+  pname = "dns_lexicon";
+  version = "3.14.1";
+  format = "pyproject";
+
+  nativeBuildInputs = [ poetry-core ];
+
+  propagatedBuildInputs = [
+    beautifulsoup4
+    cryptography
+    importlib-metadata
+    pyyaml
+    requests
+    tldextract
+  ];
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-+h1xt5vH49BWsKqxoRFg1X18Uk6qUPQwNVLWr/L/5RQ=";
+  };
+
+  meta = with lib; {
+    description = "Manipulate DNS records on various DNS providers in a standardized way.";
+    homepage = "https://github.com/AnalogJ/lexicon";
```

```suggestion
    homepage = "https://github.com/AnalogJ/lexicon";
    changelog = "https://github.com/AnalogJ/lexicon/blob/v${version]/CHANGELOG.md";
```
Would you please add meta.changelog for future updates?

---

Done, thank you.